api/lib/domain/models/Assessment.js

```diff
@@ -95,6 +96,10 @@ class Assessment {
     return this.state === Assessment.states.ENDED_BY_SUPERVISOR;
   }
 
+  isEndedByFinalization() {
```

Comment:
**nitpick**
Je pense qu'en anglais ça se dit
```suggestion
  hasBeenEndedByFinalization() {
```

Reply:
Les deux sont valables:
- isEnded : Est il terminé
- hasBeenEnded: A t il été terminé

Reply:
Ah oui tiens, au temps pour moi
https://www.linguee.fr/anglais-francais/traduction/is+ended.html